.rubocop.yml

```diff
@@ -87,7 +87,7 @@ Style/ZeroLengthPredicate:
   Enabled: false
 
 Layout/LineLength:
-  Max: 120
+  Max: 125
```

Comment:
80 and 120 are a pretty standard line lengths. Why the change to 125?

Reply:
I can change it back to 120, I mainly just tweaked it because one of the lines that had to change ended up being like 122 and Rubocop was giving me grief.